README.md

```diff
@@ -28,37 +28,37 @@ Download [the JARs][]. Or for Maven with non-Android, add to your `pom.xml`:
 <dependency>
   <groupId>io.grpc</groupId>
   <artifactId>grpc-netty</artifactId>
-  <version>1.2.0</version>
+  <version>1.3.0</version>
```

Comment:
The README change was supposed to be in a different commit so we could cherry-pick to master. I guess we just won't do that this time.